bayes/branch_and_bound/heuristic/grasp.h

```diff
@@ -0,0 +1,40 @@
+#ifndef _L2RF_BAYES_BRANCH_AND_BOUND_HEURISTIC_GRASP_H_
+#define _L2RF_BAYES_BRANCH_AND_BOUND_HEURISTIC_GRASP_H_
+
+#include <vector>
+#include "../graph.h"
+#include "../variable.h"
+#include "../inverted_index.h"
+
+namespace bayes {
+namespace branch_and_bound {
+namespace heuristic {
+
+class Grasp {
```

Comment:
Take a look at the [Google style guide](https://google.github.io/styleguide/cppguide.html) It may not be the best guide out there, but its the one we are currently using.